inst/include/Rcpp/exceptions_impl.h

```diff
@@ -0,0 +1,88 @@
+// exceptions_impl.h: Rcpp R/C++ interface class library -- exceptions
+//
+// Copyright (C) 2020  Dirk Eddelbuettel and Romain Francois
```

Comment:
Well you wrote it, so might as well be your name...

Reply:
I mostly just copy & pasted code, but sure, if you say so.